core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -184,6 +187,13 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
           error("whitelist must be specified when using new consumer in mirror maker.")
           System.exit(1)
         }
+
+        if (!consumerProps.keySet().contains(NewConsumerConfig.PARTITION_ASSIGNMENT_STRATEGY_CONFIG))
+          System.err.println("WARNING: The default partition assignment strategy of the new-consumer-based mirror maker will " +
+            "change from 'range' to 'roundrobin' in an upcoming release (so that a better load balancing can be achieved). " +
+            "If you prefer to make this switch in advance of that release add the following to the corresponding new-" +
+            "consumer config:\n" +
+            "* partition.assignment.strategy=org.apache.kafka.clients.consumer.RoundRobinAssignor")
```

Comment:
It looks a little weird when I see the output on the command line. Maybe just keep it on the same line and surround it with quotes? Not a big deal though.